### ISSUE TITLE
Android Tablet Compatibility

### ISSUE THREAD

**Issue by virgafox** [ethz-let/moodle-qtype_freehanddrawing#5]
Would it be possibile to disable page scrolling in the drawing area in order to allow "free-finger" drawing on tablets? Thanks

**Comment by virgafox** [comment]
wow have you tested it yet?

**Comment by ghost** [comment]
Only perfunctorily.

**Comment by virgafox** [comment]
Can I test this by simply replacing that file via ftp on my moodle? no need to reinstall the whole plugin?

**Comment by ghost** [comment]
AFAIK yes.

**Comment by virgafox** [comment]
I'm testing on an Android 4.2.2 both with default browser and Chrome. The page scrolling is correctly disabled in the drawing area now, but it seems that I can't draw anything. Swiping the finger nothing appears. Does it work in your little tests? thanks

**Comment by ghost** [comment]
Yes I used iOS 7.1 (both simulator and an actual iPad). I can check later if we could invest more time into Android testing...

**Comment by virgafox** [comment]
thank you very much

**Comment by virgafox** [comment]
I think that the problem on Android can be caused by this: http://uihacker.blogspot.tw/2011/01/android-touchmove-event-bug.html

**Comment by ghost** [comment]
Hi, we are a bit back tracked at the moment sorry for not responding earlier... can't really say when we'll tend to this.

**Comment by virgafox** [comment]
Hi! thanks for responding. It seems to be a very banal issue with a very banal solution from that article, but I'm not skilled in javascript, so I don't know where to put that little fix. I will try to understand the whole thing and if I'll manage to solve it I will do a pull request! bye :)

**Comment by ghost** [comment]
Hey, interestingly enough, what the article suggests to do in order to fix the problem is exactly what we do _anyway_ in order to prevent scrolling, so there is nothing _more_ to add. I guess this is not what causes the problem.

**Comment by virgafox** [comment]
yes I've noticed it, no idea how to solve.. thanks anyway
